packages/primitives/image/src/Root.web.tsx

```diff
@@ -8,7 +8,9 @@ export const Image = component(
     objectFit: 'cover',
     borderRadius: 0,
   }),
-  mapProps(({ borderRadius }) => ({ borderRadius: `${borderRadius}px` }))
+  mapWithProps(({ borderRadius }) => ({
```

Comment:
@DonGideon 👆 `mapProps` just replaces all the input props, it doesn't extend, that's why TS was complaining about everything below it. `mapWithProps` is the way to go.

Reply:
mapWithProps - done

Reply:
mapWithProps - done